templates/events/display.html

```diff
@@ -108,19 +142,25 @@ <h5 class="q-mt-none">Buy Ticket</h5>
         dismissMsg()
 
         clearInterval(paymentChecker)
-        setTimeout(function () { }, 10000)
+        setTimeout(function () {}, 10000)
       },
+      nameValidation(val) {
+        let regex = /[`!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/g
+        return (
+          !regex.test(val) ||
+          'Please enter valid name. No special character allowed.'
+        )
+      },
+      emailValidation(val) {
+        let regex = /\//g
```

Comment:
- is everything allowed? what does this regex do?

Reply:
everything allowed except `/` ! We use the name and email strings in the URL to get an invoice (should probably be a POST and send that info on data...) and the `/` doesn't work as it creates a different path!

Reply:
should i refactor it to be a POST? maybe if users are already using the extension with API, that change would break the calls!

Reply:
> everything allowed except `/`

ok, but that is not a valid email address (eg `aa 33 @@ ## ee 3`)

Reply:
pydantic will not allow it!

check this one: https://github.com/lnbits/events/pull/20/commits/97b6eed16a21c496cfb76be8b051127a24f3bb64

Reply:
ok, can you please rename it to something like `queryParamValidation` or use the [email regex we already have in core](https://github.com/lnbits/lnbits/blob/dev/lnbits/helpers.py#L176):
`[A-Za-z0-9\._%+-]+@[A-Za-z0-9\.-]+\.[A-Za-z]{2,63}`